services/src/main/java/org/keycloak/services/resources/ThemeResource.java

```diff
@@ -153,6 +154,14 @@ public Response getLocalizationTexts(@PathParam("realm") String realmName, @Quer
 
         return Cors.builder().allowedOrigins("*").auth().add(Response.ok(result));
     }
+
+    private static Theme.Type getThemeType(String themeType) {
+        try {
+            return Theme.Type.valueOf(themeType.toUpperCase());
+        } catch (IllegalArgumentException iae) {
+            return null;
```

Comment:
Seems like a good place to use an optional, but this works fine too.

Reply:
Done! It's more or less the same. :smile: